src/network.cpp

```diff
@@ -399,8 +517,20 @@ namespace xiloader
                 closesocket(sock->s);
                 sock->s = INVALID_SOCKET;
                 return false;
-            }
-        }
+
+            // Gap for previously used command codes
+
+            case 0x000A:
```

Comment:
Wrap the contents of the case with {} and break at the end/after the case

Reply:
(even if the block returns)

Reply:
done